### ISSUE TITLE
A11y_Playwright.dev_ScreenReader: Narrator is not announcing "Expanded/Collapsed" State for the "Get Started", "Playwright Test", "Guides", "Migration" and "Integrations" present in the Right Navigation section.

### ISSUE THREAD

**Issue by Raghu-addepalli** [microsoft/playwright.dev#1265]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS;#A11yTCS;#Playwright releases;#P1_Playwright.dev_Web_Jan2024;#A11ySev3;#AI Limited;#ChromiumEdge;#DesktopWeb;#FTP;#WCAG4.1.2;#Name Role Value;#Narrator;#Win11;#P1_WCP;#A11YCAIOH;#CEACCApproved

### Environmental Details:
App Name: Playwright.dev
#URL: https://playwright.dev/docs/input

### OS & Browser Details:
Microsoft Windows 11 Enterprise
Version 22621.3007
Microsoft Edge
Version 120.0.2210.144 (Official build) (64-bit)
Chrome Version: Version 120.0.6099.225 (Official Build) (64-bit)
NVDA Version: 2022.4

### Repro Steps:

1. Hit the URL: https://playwright.dev/docs/input
2. Turn on Narrator and TAB to "Get Started", "Playwright Test", "Guides", "Migration" and "Integrations" present in the Right Navigation section.
3. Now check that Narrator is not announcing "Expanded/Collapsed" State or not.

### Actual results:
Narrator is not announcing "Expanded/Collapsed" State for the "Get Started", "Playwright Test", "Guides", "Migration" and "Integrations" present in the Right Navigation section.

**Note: Issue is not Observed with NVDA.**
**Similar issue is also observed in below links.**

https://playwright.dev/docs/test-assertions
https://playwright.dev/docs/running-tests
https://playwright.dev/docs/locators
https://playwright.dev/docs/api/class-locator
https://playwright.dev/docs/writing-tests
https://playwright.dev/docs/api/class-playwright
https://playwright.dev/docs/intro
https://playwright.dev/docs/api/class-page

### Expected results:
Narrator should announce "Expanded/Collapsed" State for the "Get Started", "Playwright Test", "Guides", "Migration" and "Integrations" present in the Right Navigation section when Narrator focus lands on them.

### User Impact:
Screen Reader Users will know about the state of the controls if state is not announced by screen reader.

### Attachments:
[1265_A11y_Playwright.dev_ScreenReader.zip…]()

**Comment by mxschmitt** [comment]
Looks related to https://github.com/microsoft/fluentui/issues/11367#issuecomment-561932820.

When enabling the [advanced help text](https://www.tenforums.com/tutorials/121734-turn-off-narrator-auto-read-advanced-info-windows-10-a.html), it will read the information for me when using narrator.

Seems like there is no action item on our side, it sounds either like a bug or an intended change from the narrator team.

**Comment by Raghu-addepalli** [comment]
As per the Discussion in Office Hours call lowering this bug to Sev3

**Comment by mxschmitt** [comment]
This should be fixed.

**Comment by Raghu-addepalli** [comment]
Verified the issue in below environment issue is Fixed hence closing the bug and added attachment.

https://playwright.dev/docs/input


https://github.com/microsoft/playwright.dev/assets/91586743/b80221d7-a702-4df1-bcb9-a9e45c04df64


https://github.com/microsoft/playwright.dev/assets/91586743/c63e2159-695f-422a-a32c-719ea8ddfa88
